rpm/ndb/package.go

```diff
@@ -67,6 +68,18 @@ func (db *PackageDB) Parse(r io.ReaderAt) error {
 	return nil
 }
 
+// Headers returns an iterator over all the header blobs in the database.
```

Comment:
Nit: Should `AllHeaders()` call `Headers()`? I suppose `slices.Collect` won't work with a `Seq2`

Reply:
Ahh, I just saw `tryCollect()` worth making them all consistent?